packages/common/src/lib/common/password.ts

```diff
@@ -29,27 +29,28 @@
  *
  */
 
-/* jshint -W097 */
-/* jshint strict: false */
-/* jslint node: true */
-'use strict';
+import * as crypto from 'crypto';
+let versionOk: boolean;
 
-const crypto = require('crypto');
-let version = null;
-
-const password = pw => {
+export const password = (
+    pw: string
+): {
+    complexity: (password: string, callback: (isComplex: boolean) => void) => boolean;
+    check: (hashedPassword: string, callback: (err?: Error | null, isOk?: boolean) => void) => void;
+    hash: (salt: string, iterations: number, callback: (err?: Error | null, hash?: string) => void) => void;
+} => {
```

Comment:
The return type is complex enough to factor out into an `interface`.
Also, `@types/iobroker` has a helper type for our common error-result-callbacks (`GenericCallback<returntype>`) that you could use here.